File: AnDsSistema/2023 Semestre 1/4a-OOP/Aula02/Livro.cs
namespace Aula02
{
    public class Livro
    {
        public int Id{get; private set;}
        public string ISBN { get; private set; }
        public string Titulo { get; private set; }
        public double Preco { get; set; }

        public Livro(int id, string isbn, string titulo, double preco=0){
            this.Id = id;
            this.ISBN = isbn;
            this.Titulo = titulo;
            this.Preco = preco;
        }
    }
}

File: AnDsSistema/2023 Semestre 1/3a-Lab/aula10/exerciciosAula1-5/src/Livro.java
public class Livro {

    String titulo;
    String autor;
    double preco;

    public String getTitulo(){

        return titulo;
    }
    public void setTitulo(String titulo){

        this.titulo = titulo;
    }

    public String getAutor(){

        return autor;
    }
    public void setAutor(String autor){

        this.autor = autor;
    }

    public double getPreco(){

        return preco;
    }
    public void setPreco(double preco){

        this.preco = preco;
    }

    public Livro(String _titulo, String _autor, double _preco){

        this.titulo = _titulo;
        this.autor = _autor;
        this.preco = _preco;
    }

    public Livro(String _titulo, String _autor){

        this.titulo = _titulo;
        this.autor = _autor;
        this.preco = 100;
    }
}
